src/flightRecorder.cpp

```diff
@@ -1020,8 +1020,8 @@ class Recording {
 
     void writeLogLevels(Buffer* buf) {
         buf->putVar32(T_LOG_LEVEL);
-        buf->putVar32(LOG_ERROR - LOG_TRACE + 1);
-        for (int i = LOG_TRACE; i <= LOG_ERROR; i++) {
+        buf->putVar32(LOG_ERROR - FlightRecorder::MIN_LOG_LEVEL + 1);
+        for (int i = FlightRecorder::MIN_LOG_LEVEL; i <= LOG_ERROR; i++) {
```

Comment:
Leave the code as it was. There's nothing bad in listing all available levels, especially since there is a (theoretical) possibility to call `writeLog` directly bypassing checks in `Log::log`.